packages/ove-app-images/README.md

```diff
@@ -0,0 +1,39 @@
+# Images App
+
+This app supports visualisation of images using the OVE framework. It is based on [OpenSeadragon](https://openseadragon.github.io/) and supports high resolution zoomable images. These images can be of any format OpenSeadragon supports such as [DZI](https://openseadragon.github.io/examples/tilesource-dzi/) or [Simple Image](https://openseadragon.github.io/examples/tilesource-image/).
```

Comment:
This app supports ~~visualisation~~ **the display** of images using the OVE framework.